pkg/pipeline/transform/transform_network.go

```diff
@@ -148,6 +148,7 @@ func fillInK8s(outputEntry config.GenericMap, rule api.NetworkTransformRule) {
 				outputEntry[rule.Output+"_HostName"] = kubeInfo.HostName
 			}
 		}
+		fillInK8sZone(outputEntry, rule, *kubeInfo)
```

Comment:
We should also manage the open telemetry case in the else statement below this line.

- In case of a node we can add `k8s.node.zone`
- Other cases `k8s.host.zone`

Maybe filling the zones directly inside `fetchInformers` function in `kubernetes.go` file would actually be better ?
You can pass the rule to it and query the info accordingly.

Then in `transform_network.go` you conditionally fill the output as same as what's done for `HostIP` and `HostName`

WDYT ?

Reply:
I thought about adding this in fetchInformers, but this mean querying the informer for nodes for each flow even when the feature is disabled.

I prefered keeping it separated for performances reasons.

Reply:
About telemetry this is now supported with a uniq tag: `k8s.zone`